src/helper/blob-tools/blob.js

```diff
@@ -142,11 +141,9 @@ class Blobbiness {
                 blob.mergeBrush(lastPath);
             }
 
-            blob.cursorPreview.visible = false;
+            blob.cursorPreview.remove();
+            blob.cursorPreview = null;
             blob.onUpdateSvg();
-            blob.cursorPreview.visible = true;
-            blob.cursorPreview.bringToFront();
-            blob.cursorPreview.position = event.point;
```

Comment:
Is this commit related to the brush performance? Or is it a separate bug? If it is a separate bug, could you file a PR separately so that reviewing can be more linear? If it is a bug that came from something else you introduced here, it would be very helpful for reviewing to squash this commit with the commit that introduced the issue, or to otherwise mention in the commit message what this is addressing? It is hard to know exactly what is going on with this change.

Reply:
Sorry. This was a bug I noticed while working on performance, but isn't very related. I've pulled it out into https://github.com/LLK/scratch-paint/pull/316